src/internal/connector/exchange/data_collections_test.go

```diff
@@ -239,8 +239,11 @@ func (suite *DataCollectionsIntegrationSuite) TestMailFetch() {
 		userID    = tester.M365UserID(suite.T())
 		users     = []string{userID}
 		acct, err = tester.NewM365Account(suite.T()).M365Config()
+		ss        = selectors.Selector{}
 	)
 
+	ss.SetDiscreteOwnerIDName(userID, userID)
```

Comment:
can this use different values or will that tests stuff until we switch over?

Reply:
It could use different values.  Also, honestly I was exhausted while writing this and wanted the fasted way to a solution.  I already have a minor refactor planned for today that should add in mocks for testing.